imap_processing/codice/codice_l1a.py

```diff
@@ -116,15 +117,15 @@ def create_science_dataset(self, packets: list) -> xr.Dataset:
         )
 
         # Create a data variable for each species
-        for species_data, name in zip(self.data, self.species_names):
+        for variable_data, name in zip(self.data, self.variable_names):
```

Comment:
I changed the name here as to be more generalized and allow to iterate over priority data in addition to species data.